videos.rst

```diff
@@ -115,7 +115,25 @@ Videos and links
 
   .. raw:: html
 
-      <iframe width="560" height="315" src="https://www.youtube.com/watch?v=aqQT3J16OxI" frameborder="0" allowfullscreen>
+      <iframe width="560" height="315" src="https://www.youtube.com/embed/aqQT3J16OxI" frameborder="0" allowfullscreen>
+      </iframe>
+
+      <br/><br/>
+
+- CppCon 2019: "Why Conan? 5 reasons to use Conan package manager" by Diego Rodriguez-Losada
```

Comment:
I would exclude this one. The reason is that this is the Conan documentation. Examples, use cases, technical things, etc., are ok in the docs. This is more like a "motivational" (lets say it is more marketing than technical docs) talk, so I don't think this belongs here.

Reply:
Fair point, fixed!